3-extra/1-guess-the-number-project/script.js

```diff
@@ -1,16 +1,37 @@
 let randomNumber = Math.floor(Math.random() * 100 + 1);
 
+let messageGuess = document.querySelector(".final-output");
+let triesGuess = document.querySelector(".Tries-output");
+let countTries = 7;
+
 function guessNumber() {
   //Collect input from the user
-  let guess = document.querySelector(".inputs-Values").value;
-
-  //If the user inputs a bad input ie 0, empty string, number greater that 100, number less than zero Print "Please enter a number between 1 and 100"
+  let guessNumber = document.querySelector(".inputs-Values").value;
 
   //If the users guess is higher than the random number print Number is too high, try again (hint use .final-out class to print)
+  if (guessNumber > randomNumber && guessNumber <= 100) {
+    messageGuess.innerText = "Number is too high, try again";
+    //If the users guess is lower than the random number print Number is too low, try again  (hint use .final-out class to print)
+  } else if (guessNumber < randomNumber && guessNumber > 0) {
+    messageGuess.innerText = "Number is too low, try again";
+    //If the user has guessed the random number correctly print out the randomNumber with a message "Guess is correct. You win!"
+  } else if (guessNumber == randomNumber) {
+    messageGuess.innerText = "Guess is correct. You win!";
+    //If the user inputs a bad input ie 0, empty string, number greater that 100, number less than zero Print "Please enter a number between 1 and 100"
+  } else {
+    messageGuess.innerText = "Please enter a number between 1 and 100";
+  }
 
-  //If the users guess is lower than the random number print Number is too low, try again  (hint use .final-out class to print)
-
-  //If the user has guessed the random number correctly print out the randomNumber with a message "Guess is correct. You win!"
+  if (
+    countTries > 1 &&
+    messageGuess.innerText != "Please enter a number between 1 and 100"
+  ) {
+    countTries--;
+    triesGuess.innerHTML = `Number of Tries: ${countTries}`;
+  } else if (countTries === 0) {
+    triesGuess.innerHTML = `You Lose, the number was ${randomNumber}`;
```

Comment:
You need to recheck your logic - the "You lose.." message does not appear when the user runs out of tries.